dds/DCPS/RTPS/BaseMessageUtils.h

```diff
@@ -154,15 +154,11 @@ void message_block_to_sequence(const ACE_Message_Block& mb_locator, T& out)
   std::memcpy (out.get_buffer(), mb_locator.rd_ptr(), mb_locator.length());
 }
 
-#ifndef OPENDDS_SAFETY_PROFILE
```

Comment:
That's gonna conflict with the operator safety profile generates.

Reply:
I was wondering why that was in there, I didn't realize the safety profile build made its own assignment operators. Regardless, the safety profile builds were failing so this is still a problem, just need to figure out what the right solution is. Now if only azure pipelines wouldn't crap out in the prep stage or would correctly handle reruns...